homeassistant/components/image_processing/__init__.py

```diff
@@ -262,7 +262,7 @@ def async_process_faces(self, faces: list[FaceInformation], total: int) -> None:
                 continue
 
             face.update({ATTR_ENTITY_ID: self.entity_id})
-            self.hass.async_add_job(self.hass.bus.async_fire, EVENT_DETECT_FACE, face)
+            self.hass.bus.async_fire(EVENT_DETECT_FACE, dict(face))
```

Comment:
This is likely changing the functionality.

I'd add a type ignore for now instead

Reply:
Make sense, changed